Add YAML description meta tags (#2582)
meilisearch/documentation: learn/cookbooks/postman_collection.mdx

---

> description: Postman is a platform that lets you create HTTP requests you can easily reuse and share with everyone. This how-to guide explains how to use Postman to help testing and debugging Meilisearch's API.

Author: ```suggestion
description: This how-to guide explains how to use Postman when testing and debugging Meilisearch's API.
```
